robot/client/client.go

```diff
@@ -1416,7 +1459,7 @@ func unaryClientInterceptor() googlegrpc.UnaryClientInterceptor {
 	) error {
 		md := robot.Version
 		stringMd := fmt.Sprintf("go;%s;%s", md.Version, md.APIVersion)
-		ctx = metadata.AppendToOutgoingContext(ctx, "viam_client", stringMd)
```

Comment:
The server-side of this `viam_client` SDK metadata feature was missing, so I added it.